GiTinder/Controllers/GiTinderContex.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Threading.Tasks;
+using GiTinder.Models;
+
+namespace GiTinder.Controllers
+{
+    public class GiTinderContex
+    {
+        public object Settings { get; internal set; }
```

Comment:
@FilipTeply this class should extend dbcontext and settings property should be dbset